frontend/src/metabase/admin/settings/components/widgets/ModelCachingScheduleWidget/ModelCachingScheduleWidget.jsx

```diff
@@ -41,14 +41,15 @@ const PersistedModelRefreshIntervalWidget = ({
   const [customCronSchedule, setCustomCronSchedule] = useState(
     // We don't allow to specify the "year" component, but it's present in the value
     // So we need to cut it visually to avoid confusion
-    isCustom ? formatCronExpression(setting.value) : "",
+    isCustom ? formatCronExpression(setting.value || setting.default) : "",
```

Comment:
Maybe `setting.value ?? setting.default`? Or it will always use `default` when the value is `false` (which might be ok here but not sure if it was intended)

Reply:
I'm pretty sure in this case, `setting.value` should only be a string or null, but it doesn't hurt to be more specific! Changed.